### ISSUE TITLE
Wiki still pointing to google code

### ISSUE THREAD

**Issue by juszczakn** [lispbuilder/lispbuilder#64]
I noticed that links in the [Wiki User Guide](https://github.com/lispbuilder/lispbuilder/wiki/UsingLispbuilderSDL) are still pointing to google code. It doesn't appear to be open for editing, so wondering if someone could update it? 

In general there appears to be a bit of bitrot from the gcode transition, might be good to give it a once over.

**Comment by elliottslaughter** [comment]
Good catch. If you are interested in making a patch for this, I'd be happy to accept it. (Unfortunately Github doesn't accept pull requests to wikis, so I think you'd have to upload it as a diff.)

You can clone the wiki with:

```
git clone https://github.com/lispbuilder/lispbuilder.wiki.git
```

Thanks for catching this!

**Comment by priyadarshan** [comment]
Came here to say the same. I could try to do the Merge Request myself.  Where can one find the correct link, that is, the proper source for LISPBUILDER-SDL?

**Comment by elliottslaughter** [comment]
You can clone with `git clone https://github.com/lispbuilder/lispbuilder.wiki.git`.

I don't think there an ability to issue pull requests to the wiki directly, but you can upload the diff to this issue (as a `.txt` file), or you can do something like the steps suggested below:

https://www.growingwiththeweb.com/2016/07/enabling-pull-requests-on-github-wikis.html

Thanks!

**Comment by priyadarshan** [comment]
Thank you, I will do that.

May I have the the working link, that is, the new place instead of http://code.google.com/p/lispbuilder/downloads/ ?

**Comment by elliottslaughter** [comment]
Releases are at https://github.com/lispbuilder/lispbuilder/releases . But I think the vast majority of users download via Quicklisp anyway these days, we should probably point people towards that. There were copies of SDL binaries and such, that we haven't put back up (these may be lost, I'd have to check), but hopefully people can get those from package managers or from SDL proper.

**Comment by priyadarshan** [comment]
Thank you, I will try to submit PR by tomorrow.

**Comment by priyadarshan** [comment]
Unfortunately, forking the repository will not fork the wiki too :(

Would you consider moving the documentation to its own place, like `/doc`so that it would be easier for people to submit changes/improvements? Quite a few project likes to do that, becasue any change to documentation would be part of the main repository.

**Comment by elliottslaughter** [comment]
Ok, let's try this. Can you send the PR to this repo?

https://github.com/lispbuilder/wiki

Thanks.

**Comment by priyadarshan** [comment]
See https://github.com/lispbuilder/wiki/pull/1#issuecomment-508815268

**Comment by elliottslaughter** [comment]
Merged the PR, please let me know if there is anything else.

**Comment by priyadarshan** [comment]
Yes, I will. Thank you so much for considering this one for now, and thank you for Lispbuilder!
